app/views/teachers/students/index.html.erb

```diff
@@ -1,17 +1,30 @@
-<div id="students">
+<div id="students" class="container pt-5">
 
-  <% @students.each do |student| %>
-    <%= render "teachers/students/student", student: student %>
-    <p>
-      <%# TODO: SHOW entire history of this user %>
-      <%= link_to "Show this student", teacher_student_url(@user, student) %>
-      <br/>
-      <%= link_to "Edit this student", edit_teacher_student_url(@user, student) %>
-    </p>
-    <br />
-  <% end %>
+  <h2 class="title">Enrolled students<h2>
 
-  <%= will_paginate @students, renderer: RemoteLinkPagination::LinkRenderer, :class=>"" %>
-  <%= link_to "Back to dashboard", teacher_url(@user) %>
+  <table class="table is-striped is-hoverable is-fullwidth">
+    <thead>
+      <tr>
+        <th>Student name</th>
+        <th>Belt</th>
+        <th>Group</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @students.each do |student| %>
+        <tr>
+          <%= render "teachers/students/student", student: student, teacher: @user %>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
 
+  <%= will_paginate @students, renderer: RemoteLinkPagination::LinkRenderer, :class=>"is-flex is-justify-content-space-around" %>
```

Comment:
```suggestion
  <%= will_paginate @students, renderer: RemoteLinkPagination::LinkRenderer, class: "is-flex is-justify-content-space-around" %>
```

Reply:
@bryanmg why is this marked as resolved?